test/hotspot/jtreg/compiler/vectorapi/reshape/utils/TestCastMethods.java

```diff
@@ -30,16 +30,15 @@
 import static compiler.vectorapi.reshape.utils.VectorSpeciesPair.makePair;
 
 /**
- * The cast intrinsics implemented on each platform, commented out tests are the ones that are
- * supposed to work but currently don't.
```

Comment:
Maybe you can add again the reason for the commented out `makePair()` calls.

Reply:
Address comments, thanks a lot.